src/server/config.rs

```diff
@@ -19,7 +19,7 @@ const DEFAULT_STATUS_ADDR: &str = "127.0.0.1:20180";
 const DEFAULT_GRPC_CONCURRENCY: usize = 4;
 const DEFAULT_GRPC_CONCURRENT_STREAM: i32 = 1024;
 const DEFAULT_GRPC_RAFT_CONN_NUM: usize = 1;
-const DEFAULT_GRPC_MEMORY_POOL_QUOTA: u64 = isize::MAX as u64;
+const DEFAULT_GRPC_MEMORY_POOL_QUOTA_RATIO: f64 = 0.1;
```

Comment:
Is this too large?

Reply:
I think `cmp::max(total * 0.1, 2GB)` is ok.